smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/integration/HttpProtocolServiceClient.kt

```diff
@@ -256,17 +271,7 @@ open class HttpProtocolServiceClient(
         }
     }
 
-    // Renders the passed properties using the passed block to generate text,
-    // optionally comma-separating the lines as they are rendered.
-    protected fun renderProperties(
-        properties: List<ConfigProperty>,
-        commaSeparated: Boolean,
-        block: (ConfigProperty) -> String
-    ) {
-        val commaOrBlank = ",".takeIf { commaSeparated } ?: ""
-        properties.forEach { property ->
-            val separator = commaOrBlank.takeIf { property != properties.last() } ?: ""
-            writer.write("\$L\$L", block(property), separator)
-        }
```

Comment:
The function above is removed.  Properties are now rendered inline by using the writer directly instead.